app/core/vectordb/chroma.py

```diff
@@ -95,26 +84,28 @@ def add_to_collection(self, docs: List[schema.Document], **kwargs) -> None:
                 {
                     "label": doc.label,
                     "media": ",".join(doc.media),
-                    "links": ",".join(doc.links),
+                    "links": ",".join(str(doc.links)),
```

Comment:
I'm curious how you can have a string parameter to `join()`...

Reply:
Probably a pydantic suggested change. Shall check this

Reply:
Made a change for this. Our existing tests seems to be working in both cases though.